eng/common/scripts/Package-Properties.ps1

```diff
@@ -15,7 +15,7 @@ class PackageProps
     [boolean]$IsNewSdk
     [string]$ArtifactName
     [string]$ReleaseStatus
-    [string[]]$DependentPackages
+    [string[]]$CanaryPackages
```

Comment:
Can we name this something like "AdditionalValidationPackages" or something a little meaningful to explain what they are intended for?

Reply:
That's what the `CanaryPackages` rename was supposed to be 👍 

If that's not clear enough then sure, I'll go with your name.